cli/azd/pkg/templates/source_manager.go

```diff
@@ -66,14 +68,15 @@ type SourceManager interface {
 	// Remove removes a template source.
 	Remove(ctx context.Context, name string) error
 	// CreateSource creates a new template source from a source configuration
-	CreateSource(ctx context.Context, source *SourceConfig) (Source, error)
+	CreateSource(ctx context.Context, source *SourceConfig, console input.Console) (Source, error)
```

Comment:
Again - I would really advise that creating a source does NOT require a reference to `input.Console`.